test/integration/domain/test_artifact.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from clarity_ext.domain.artifact import StepInputOutputMapRepository
+from clarity_ext.clarity import ClaritySession
+import itertools
+
+
```

Comment:
In future, we could maybe have a separate database for integration tests? In the long run it will be hard to keep track of which of the steps that shouldn't be updated through the ui and not.